evals/label_embedding_interpolation_eval.py

```diff
@@ -0,0 +1,262 @@
+    batch_dynamics = get_maestro_record(query[1])
+
+    # grab only name without extension
+    filename_pitch = batch_pitch["filename"]
+    filename_dynamics = batch_dynamics["filename"]
+
+    filename = f"pitch: {filename_pitch}, dynamics: {filename_dynamics}"
+
+    # unpack other attributes
+    pitch = batch_pitch["pitch"].to(cfg.train.device)
+    dstart = batch_dynamics["dstart"].numpy()
+    duration = batch_dynamics["duration"].numpy()
+    velocity = batch_dynamics["velocity"]
+
+    velocity_bin = batch_dynamics["velocity_bin"].to(cfg.train.device)
+    dstart_bin = batch_dynamics["dstart_bin"].to(cfg.train.device)
+    duration_bin = batch_dynamics["duration_bin"].to(cfg.train.device)
+
+    if conditioning_models is not None:
+        with torch.no_grad():
+            velocity_time_conditioning_model, pitch_conditioning_model = conditioning_models
+
+            label_emb_dynamics = velocity_time_conditioning_model(velocity_bin, dstart_bin, duration_bin)
+            label_emb_pitch = pitch_conditioning_model(pitch)
+            conditioning_embeding = torch.lerp(label_emb_dynamics, label_emb_pitch, interpolation_weight)
+    else:
+        conditioning_embeding = None
+
+    noise = torch.randn(velocity.size()).to(cfg.train.device)
+
+    # sample velocities from standard and ema model
+    fake_velocity = gen.sample(
+        x=noise,
+        conditioning_embeding=conditioning_embeding,
+        intermediate_outputs=False,
+        classifier_free_guidance_scale=classifier_free_guidance_scale,
+    )
+    fake_velocity_ema = gen_ema.sample(
+        x=noise,
+        conditioning_embeding=conditioning_embeding,
+        intermediate_outputs=False,
+        classifier_free_guidance_scale=classifier_free_guidance_scale,
+    )
+
+    fake_velocity = torch.clip(fake_velocity, -1, 1)
+    fake_velocity_ema = torch.clip(fake_velocity_ema, -1, 1)
+
+    pitch = pitch[0].cpu().numpy()
+
+    velocity = denormalize_velocity(velocity[0][0].numpy())
+    fake_velocity = denormalize_velocity(fake_velocity[0][0].cpu().numpy())
+    fake_velocity_ema = denormalize_velocity(fake_velocity_ema[0][0].cpu().numpy())
+
+    original_piece = to_midi_piece(pitch, dstart, duration, velocity)
+    model_piece = to_midi_piece(pitch, dstart, duration, fake_velocity)
+    model_ema_piece = to_midi_piece(pitch, dstart, duration, fake_velocity_ema)
+
+    original_midi = original_piece.to_midi()
+    model_midi = model_piece.to_midi()
+    model_ema_midi = model_ema_piece.to_midi()
+
+    # save as mp3
+    render_midi_to_mp3(original_midi, f"tmp/mp3/{filename}-simple-interpolation.mp3")
+    render_midi_to_mp3(model_midi, f"tmp/mp3/{filename}-model.mp3")
+    render_midi_to_mp3(model_ema_midi, f"tmp/mp3/{filename}-model-ema.mp3")
+
+    # save as midi
+    original_midi.write(f"tmp/midi/{filename}-simple-interpolation.midi")
+    model_midi.write(f"tmp/midi/{filename}-model.midi")
+    model_ema_midi.write(f"tmp/midi/{filename}-model-ema.midi")
```

Comment:
```suggestion
    render_midi_to_mp3(original_midi, f"tmp/mp3/{filename}-original.mp3")
    render_midi_to_mp3(model_midi, f"tmp/mp3/{filename}-model.mp3")
    render_midi_to_mp3(model_ema_midi, f"tmp/mp3/{filename}-model-ema.mp3")

    # save as midi
    original_midi.write(f"tmp/midi/{filename}-original.midi")
    model_midi.write(f"tmp/midi/{filename}-model.midi")
    model_ema_midi.write(f"tmp/midi/{filename}-model-ema.midi")
```

I think that's require for consistency with the dashboard? I also think that the word "simple" should be avoided at all cost, since nothing is ever simple :)